docs/build/guides/transactions/claimable-balances.mdx

```diff
@@ -68,368 +68,623 @@ Each of these accounts can only claim the balance under unique conditions. Accou
-    check(err)
-    txResp, err := client.SubmitTransaction(tx)
-    check(err)
-
-    fmt.Println(txResp)
-    fmt.Println("Claimable balance created!")
+	return "", fmt.Errorf("no operation results")
 }
 ```
 
-```python
-import time
-from stellar_sdk.xdr import TransactionResult, OperationType
-from stellar_sdk.exceptions import NotFoundError, BadResponseError, BadRequestError
-from stellar_sdk import (
-    Keypair,
-    Network,
-    Server,
-    TransactionBuilder,
-    Transaction,
-    Asset,
-    Operation,
-    Claimant,
-    ClaimPredicate,
-    CreateClaimableBalance,
-    ClaimClaimableBalance
-)
+</CodeExample>
 
-server = Server("https://horizon-testnet.stellar.org")
+At this point, the `ClaimableBalanceEntry` exists in the ledger, but we’ll need its Balance ID to claim it. You can call the RPC's [`getLedgerEntries`](../../../data/apis/rpc/api-reference/methods/getLedgerEntries.mdx) endpoint to do this.
 
-A = Keypair.from_secret("SANRGB5VXZ52E7XDGH2BHVBFZR4S25AUQ4BR7SFXIQYT5J6W2OES2OP7")
-B = Keypair.from_public_key("GAAPSRMYNFAO3TDQTLNLKN76IQ3E6IQAKU23PSQX3BIV7RTEBXHQIWU6")
+<CodeExample>
 
-# NOTE: Proper error checks are omitted for brevity; always validate things!
+```js
+import * as StellarSdk from "@stellar/stellar-sdk";
+
+// Replace with your actual Claimable Balance ID
+// Format: 72 hex characters (includes ClaimableBalanceId type + hash)
+const BALANCE_ID =
+  "00000000db1108ff108a807150d02b8672d9a8c0e808bff918cdbe5c7605e63a7f565df5";
+
+/**
+ * Fetches and displays claimable balance details using Stellar RPC
+ */
+async function fetchClaimableBalance(balanceId) {
+  const server = new StellarSdk.rpc.Server(
+    "https://soroban-testnet.stellar.org",
+  );
 
-try:
-    aAccount = server.load_account(A.public_key)
-except NotFoundError:
-    raise Exception(f"Failed to load {A.public_key}")
+  try {
+    console.log(`Looking up balance ID: ${balanceId}`);
+
+    // Parse the claimable balance ID from hex XDR
+    const claimableBalanceId = StellarSdk.xdr.ClaimableBalanceId.fromXDR(
+      balanceId,
+      "hex",
+    );
+
+    // Create ledger key for the claimable balance entry
+    const ledgerKey = StellarSdk.xdr.LedgerKey.claimableBalance(
+      new StellarSdk.xdr.LedgerKeyClaimableBalance({
+        balanceId: claimableBalanceId,
+      }),
+    );
+
+    console.log(`Fetching from RPC server...`);
+
+    // Use SDK's getLedgerEntries method with XDR object array
+    const response = await server.getLedgerEntries(ledgerKey);
+
+    if (response.entries && response.entries.length > 0) {
+      const claimableBalance = response.entries[0].val.claimableBalance();
+
+      const asset = StellarSdk.Asset.fromOperation(claimableBalance.asset());
+
+      console.log(`Found claimable balance`);
+      console.log(`Amount: ${claimableBalance.amount().toString()}`);
+      console.log(`Asset: ${asset.toString()} `);
+      console.log(
+        `Number of claimants: ${claimableBalance.claimants().length}`,
+      );
```

Comment:
nitpick and no strong opinion but this could be omitted for brevity given the subsequent listing of claimants